### PR TITLE
Start page

### PR DESCRIPTION
**Pull request by victoriasloan**
Fully tested. No hate.

### CODE REVIEW

File: src/__tests__/App.test.js [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+  expect(app.find("input").length).toEqual(1);
+  expect(app.state("enterLocation")).toEqual(true);
+});
+
+it("Should update the state of the input when the user starts to type in their city", () => {
+  const app = shallow(<App/>);
+  app.setState({enterLocation: true})
+  app.find("input").simulate("change", {
+    target: {
+      value: "woody"
```
Comment (mmckeaveney) [inline src/__tests__/App.test.js]: alt text

File: src/components/App.js [hunk 2]
```diff
@@ -1,18 +1,50 @@
...
+  handleClick = () => {
+    this.setState({ enterLocation: true });
+  };
+
+  handleInput = () => {
+    console.log(this.state.input);
+    //go to api
+  };
+
+  updateInputState = e => {
```
Comment (mmckeaveney) [inline src/components/App.js]: Give these better names

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class Button extends Component {
```
Comment (mmckeaveney) [inline src/components/Button.js]: We probably need to describe these as ```LocationButton``` or something rather than such a generic name

Comment (victoriasloan) [reply src/components/Button.js]: sure you know yourself

**Pull request merged**